src/lib/app/lib/security-headers.ts

```diff
@@ -6,7 +6,6 @@ const securityHeaders = new Map([
   ['X-Frame-Options', 'SAMEORIGIN'],
   ['X-Content-Type-Options', 'nosniff'],
   ['Referrer-Policy', 'no-referrer'],
-  ['Content-Security-Policy', "script-src 'self'; frame-ancestors 'none';"],
```

Comment:
Question: why do you use `new Map()` instead of simple object creation?

Reply:
I just don't really trust objects at all. You know how iterating over an object might give you some unexpected properties? I don't know if that really happens or not, I just dislike objects for that reason. Maps are a natural way of storing key-value pairs, and they are rock solid, so I just like them much more